README.md

```diff
@@ -1,52 +1,63 @@
-# ibm-fortigate-terraform-deploy
-
-FortiGate IBM Deployment Template.
-
 ## Description
 
-A Terraform script to deploy a single FortiGate in IBM Cloud using the Schematics service.
-
-## Requirements
-
--   [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) 0.13+
--   FortiOS 6.4.3 BYOL License.
+Deploy a single BYOL FortiGate in IBM Cloud using the Schematics service.
 
 ## Deployment overview
 
 > **Note:** For a local deployment a Gen 2 API key will be needed. For details see: [IBM Gen 2 API key](https://cloud.ibm.com/docs/terraform?topic=terraform-provider-reference)
 
+This deployment requires that you already have the following already configured:
+
+-   A VPC
+-   Two subnets
+-   An ssh key
+
 Terraform deploys the following components:
 
--   A VPC with two subnets
 -   A FortiGate BYOL instance with two NICs, one in each subnet
--   A security group with no restrictions
 -   A Floating Public IP address attached to the FortiGate
 -   A Logging disk
--   A basic bootstrap, including license if supplied.
 
-## Deployment
+## Deployment:
+
+1. Fill in the required VPC, Subnet and ssh key values as shown below:
+
+![IBM FortiGate Deploy](https://raw.githubusercontent.com/fortinet/ibm-fortigate-terraform-deploy/main/imgs/imgs/step_6_a.png)
```

Comment:
imgs/imgs ?
https://raw.githubusercontent.com/fortinet/ibm-fortigate-terraform-deploy/main/imgs/step_6_a.png seems correct

Reply:
also a bunch of cases below